src/main/kotlin/com/kebab/user/service/dto/TokenDto.kt

```diff
@@ -0,0 +1,3 @@
+package com.kebab.user.service.dto
+
+class TokenDto(val token: String)
```

Comment:
I do not see the place, where it is used

Reply:
AuthenticationController